modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -3079,19 +3077,20 @@ private double getOrCreateCellSize (int idx, boolean create) {
      */
     void updateCellSize(T cell) {
         int cellIndex = cell.getIndex();
-        int currentIndex = computeCurrentIndex();
-        double oldOffset = computeViewportOffset(getPosition());
-
 
         if (itemSizeCache.size() > cellIndex) {
             Double oldSize = itemSizeCache.get(cellIndex);
             double newSize = isVertical() ? cell.getLayoutBounds().getHeight() : cell.getLayoutBounds().getWidth();
```

Comment:
If we have a fixed cell size set, can we save the call to get the layout bounds here?
Since this is also done above in `getOrCreateCellSize`.
And the layout bounds are calculated lazily, so this may improve the performance here as well.

Reply:
@johanvos I still think this should be evaluated.

Reply:
Perhaps you could file a follow-up JBS issue?

Reply:
Filed: https://bugs.openjdk.org/browse/JDK-8309470